web/controllers/comments.go

```diff
@@ -71,7 +71,7 @@ func CommentCreate(w http.ResponseWriter, r *http.Request) {
 		funcs.LinkFromLinks(comment.Meta.Links, "commentPage").Href,
 	)
 
-	u, err := url.Parse(`/conversations/253639/?offset=775`)
+	u, err := url.Parse(location)
```

Comment:
Good discovery of intention! :)

I'd tested in Go Playground adding the fragment and copied the working code back into here.